mlrun/feature_store/ingestion.py

```diff
@@ -93,7 +93,8 @@ def _add_data_states(
         cache.cache_table(featureset.uri, table, True)
 
     entity_columns = list(featureset.spec.entities.keys())
-    key_field = entity_columns[0] if entity_columns else None
+    key_field = entity_columns if entity_columns else None
```

Comment:
The naming and probably (and hopefully) the whole handling of it is considering one key field, not a list of them
Is this really working ? sounds like the terminology need to change
@yaronha WDYT ?

Reply:
dont think it can work just by changing this, need the source drivers to accept list, and if we change it to a list we may need to rename the storey classes and this to call it `key_fields` and accept list

Reply:
yes, the changes were already made in storey (to accept keys as list). merged into storey 0.3.8

Reply:
(i left the name of the parameter 'key_field' in order to not break API)